File: 4/04613.py
while 1:
    s = input()
    if s == '#': break

    quicksum = 0
    for i in range(len(s)):
        if s[i].isalpha():
            quicksum += (i + 1) * (ord(s[i]) - ord('A') + 1)

    print(quicksum)

File: 4/04613.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	while (1) {
		string s; getline(cin, s);
		if (s == "#") break;

		int quicksum = 0;
		for (int i = 0; i < s.length(); i++)
			if (isalpha(s[i]))
				quicksum += (i + 1) * (s[i] - 'A' + 1);

		cout << quicksum << endl;
	}
}
